Portfolio/index.html

```diff
@@ -1,209 +1,236 @@
-console.log('Hello World');
-
-
-
 <!DOCTYPE html>
 <html lang="en">
 
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="style.css">
     <title>Document</title>
 </head>
 

```

Comment:
The 'bold' tag is deprecated and should be replaced with 'strong' for semantic correctness.